composer.json

```diff
@@ -23,20 +23,20 @@
 	"repositories": [
 		{
 			"type": "vcs",
-			"url": "https://github.com/tomzx/gitter"
+			"url": "https://github.com/mallardduck/gitter"
 		}
 	],
 	"require": {
-		"php": ">=5.6.0",
-		"klaussilveira/gitter": "dev-php-semver-checker",
+		"php": ">=7.2.29",
+		"klaussilveira/gitter": "dev-7.4-update",
```

Comment:
Note: This would be reverted once this PR https://github.com/tomzx/gitter/pull/4/files is merged in.